claim/apps.py

```diff
@@ -52,6 +52,9 @@ class ClaimConfig(AppConfig):
     gql_mutation_delete_claims_perms = []
     claim_print_perms = []
     claim_attachments_root_path = None
+    claim_autogenerate_code_length = 8
```

Comment:
I'm not sure if this should be part of the config. Does Nepali use different lengths across implementations? This property seems to be specific for a particular generator. We could introduce configuration like `autogenerated_claim_code_config` that would store information about code generator setup.

Reply:
I think it could be used by different generators as well - that was my intention, length of autogenerated part of the code seems very generic to me and would be helpful in all generators. If you still think that it could be moved to a separate file, please let me know.

Reply:
On one hand it make sense, however from implementation perspective I'd say it's simpler to hardcode value in the autogenerator during development than adding the config. One the other some general-use generators could actually be configured and in that case length configuration is better option than having multiple generators that would use different code len. In that case, I think that it could be kept as a default in autogenerated_claim_code_config, i.e.: 
```python 
autogenerated_claim_code_config = { 'code_length': 8 }
```